lib/abilities.ex

```diff
@@ -9,5 +9,25 @@ defimpl Canada.Can, for: Pairmotron.User do
   def can?(%User{id: user_id}, action, %Group{owner_id: user_id})
     when action in [:edit, :update, :delete], do: true
 
+  def can?(%User{id: user_id}, :show, project = %Project{}) do
+    if is_nil(project.group_id) do
+      false
+    else
+      project = project |> Pairmotron.Repo.preload([{:group, :users}])
+      project.group.users
+      |> Enum.any?(fn user -> user.id == user_id end)
+    end
+  end
```

Comment:
If a user were to ever edit a retro that includes a project that no longer belongs to a group of the user, will the user be able to save the changes? Would we want the user to be able to save the changes?

Oh, this is handled by not allowing the user to edit the group, right?